config/config.go

```diff
@@ -74,6 +74,10 @@ func set() *Config {
 
 	// Cache purge threshold in minutes
 	v.SetDefault("FIREHOSE_CACHE_DURATION_MINS", 30)
+	// Cache instance update in seconds
+	v.SetDefault("FIREHOSE_CACHE_UPDATE_INTERVAL_SECS", 30)
```

Comment:
should we increase the default value a little bit?

Reply:
yes i will use 60 sec thanks. this should help on the api amounts of calls